### PR TITLE
implemented fast-api library

### PR DESCRIPTION
**Pull request by mariiahorbova**

### CODE REVIEW

File: alembic/README [hunk 1]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (oleksiikolii) [inline alembic/README]: ahtung no blank line

**Review by oleksiikolii — APPROVED** [review]

File: crud.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.orm import Session
+from models import Author, Book
+from schemas import AuthorCreate, BookCreate
+
+
+def get_author_by_name(db: Session, name: str):
+    return (
+        db.query(Author).filter(Author.name == name).first()
+    )
+
```
Comment (Kirontiko) [inline crud.py:10]: Aren`t we filtering by author id?
image

File: crud.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+def get_author(db: Session, author_id: int):
+    return db.query(Author).filter(Author.id == author_id).first()
+
+
+def get_book_by_title(db: Session, title: str):
+    return (
+        db.query(Book).filter(Book.title == title).first()
+    )
+
```
Comment (Kirontiko) [inline crud.py:32]: Same here
image

File: alembic/README (hunk 1 above)
Comment (Kirontiko) [inline alembic/README]: Add blank line

**Review by Kirontiko — CHANGES_REQUESTED** [review]

**Review by Kirontiko — APPROVED** [review]
Sry, didn`t see filtering by ids

File: .flake8 [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[flake8]
+inline-quotes = "
+ignore = E203, E266, W503, N807, N818, F401
+max-line-length = 79
+max-complexity = 18
+select = B,C,E,F,W,T4,B9,Q0,N8,VNE
+exclude =
+    venv
+    .env
+    **versions
```
Comment (Oleg-s-s) [inline .flake8:10]: blank line :eyes:

**Review by Oleg-s-s — APPROVED** [review]

**Pull request closed without merging**